cmd/namespace/sleep.go

```diff
@@ -29,9 +29,12 @@ func Sleep(ctx context.Context) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "sleep <name>",
 		Short: "Sleeps a namespace",
-		Args:  utils.ExactArgsAccepted(1, ""),
+		Args:  utils.MaximumNArgsAccepted(1, ""),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			nsToSleep := args[0]
+			nsToSleep := okteto.Context().Namespace
```

Comment:
What happens if the user doesn't have the `okteto ctx` set? I think this solve be moved after the ctx command run

Reply:
+1

Reply:
![image](https://user-images.githubusercontent.com/86051118/234344596-a11f4727-a76d-4423-8538-29831ffb91bd.png)

Reply:
`sleep` and `wake` command will only work for the clusters that have okteto installed.

Reply:
What I meant is that I think that if you remove `~/.okteto/context/config.json` (or have it malformed) it would throw a panic and the user won't have any output. If we move the code after the context command, the user will get the selector for choosing the cluster to connect to